test/specs/example.e2e.ts

```diff
@@ -0,0 +1,23 @@
+describe('WDIO VSCode Service', function () {
+  this.retries(3);
+
+  it('should be able to load VSCode', async () => {
+    const workbench = await browser.getWorkbench();
+    expect(await workbench.getTitleBar().getTitle()).toBe(
+      '[Extension Development Host] hubspot-cms-vscode'
+    );
+  });
+
+  it('extension should appear in sidebar', async () => {
+    const workbench = await browser.getWorkbench();
+    const viewControls = await workbench.getActivityBar().getViewControls();
+    expect(await Promise.all(viewControls.map((vc) => vc.getTitle()))).toEqual([
+      'Explorer',
+      'Search',
+      'Source Control',
+      'Run and Debug',
+      'Extensions',
+      'HubSpot',
+    ]);
```

Comment:
If someone was to change the order of their controls, would that break this?

Reply:
Just tested on my machine, order on my local VSCode instance had no impact on the one `wdio` spins up